sweat/models/dataframes.py

```diff
@@ -0,0 +1,74 @@
+from sweat.algorithms.hrm import heartrate_models
```

Comment:
As discussed in #8 this code could also move to `sweat.io.models`. Since the wdf probably won't be used directly that might make sense. But can be done later when we have a clearer view of how `sweat.io` will work/look like.

Reply:
agree